resources/views/WelcomePage/WelcomePage.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {PageHeader, Card, Row, Col, Divider, Typography} from 'antd'
+
+const {Text} = Typography
+
+const WelcomePage = () => (
+  <>
+    <PageHeader
+      className='welcome-page-header'
```

Comment:
not very sure why this `className` is needed, as there is no css file with class specified